server/src/entities/asset-face.entity.ts

```diff
@@ -3,9 +3,10 @@ import { FaceSearchEntity } from 'src/entities/face-search.entity';
 import { PersonEntity } from 'src/entities/person.entity';
 import { Column, Entity, Index, ManyToOne, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
 
-export enum SourceType {
-  MACHINE_LEARNING = 'machine-learning',
-  EXIF = 'exif',
+// eslint-disable-next-line unicorn/no-static-only-class
+export class SourceType {
+  static MACHINE_LEARNING = null;
+  static EXIF = 'exif';
 }
```

Comment:
I see what you're trying to do, but it feels a bit weird - I would just remove the `MACHINE_LEARNING` entry and make it an enum again. Also, it occurs to me that since there are different formats for this, `EXIF` might not be specific enough. Maybe `DIGIKAM` would be better?

Reply:
> I see what you're trying to do, but it feels a bit weird - I would just remove the MACHINE_LEARNING entry and make it an enum again.

The problem with enums is that cannot be nullable, aren't they? Can you share an example of what you're looking for?

> Also, it occurs to me that since there are different formats for this, EXIF might not be specific enough. Maybe DIGIKAM would be better? 

Similarly to the `ImmichTags` conversation we can use `exif` for now and create more types in the future when we cover more metadata region formats

Reply:
> The problem with enums is that cannot be nullable, aren't they? Can you share an example of what you're looking for?

I mean that the null shouldn't be part of the enum. Just use `null` directly.

Reply:
> Similarly to the ImmichTags conversation we can use exif for now and create more types in the future when we cover more metadata region formats

Yeah, that's fair.

Reply:
Something like this 0150af6113a0ad28ad0ab859b0b355f26a0a0db9?